distributed/deploy/tests/test_local.py

```diff
@@ -328,15 +328,17 @@ def test_remote_access(loop):
         sync(loop, assert_can_connect_from_everywhere_4_6, c.scheduler.port)
 
 
-def test_memory(loop):
-    with LocalCluster(scheduler_port=0, processes=False, silence_logs=False,
-                      diagnostics_port=None, loop=loop) as cluster:
+@pytest.mark.parametrize('n_workers', [None, 1, 2, 3, 4])
+def test_memory(loop, n_workers):
+    with LocalCluster(n_workers=n_workers, scheduler_port=0, processes=False,
+                      silence_logs=False, diagnostics_port=None, loop=loop) as cluster:
         assert sum(w.memory_limit for w in cluster.workers) <= TOTAL_MEMORY
 
 
-def test_memory_nanny(loop):
-    with LocalCluster(scheduler_port=0, processes=True, silence_logs=False,
-                      diagnostics_port=None, loop=loop) as cluster:
+@pytest.mark.parametrize('n_workers', [None, 1, 2, 3, 4])
+def test_memory_nanny(loop, n_workers):
```

Comment:
This test takes about a second to run, so adding the parameterization above adds something like 4s extra to the test suite.  If we can cut this down, perhaps by using two entries rather than five that would be helpful to keep tests fast.

The test above is faster because it doesn't create new processes.

Reply:
Done, thanks!